server/utils/azureAppInsights.ts

```diff
@@ -0,0 +1,57 @@
+import { config } from 'dotenv'
```

Comment:
Converted to typescript and aligned with how the [template project does things](https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/utils/azureAppInsights.ts) for consistency.

It was doing all the same stuff, just in a slightly different way